pkg/iac-providers/helm/v3/load-dir.go

```diff
@@ -192,23 +191,27 @@ func (h *HelmV3) renderChart(chartPath string, chartMap helmChartData, templateD
 		Templates: chartFiles,
 	}
 
-	var v chartutil.Values
-	v, err := chartutil.CoalesceValues(c, chartutil.Values{
-		"Values": valueMap,
-		"Release": chartutil.Values{
-			"Name": defaultChartName,
-		},
-	})
+	// create rendered values
+	options := chartutil.ReleaseOptions{
+		Name:      defaultChartName,
+		Namespace: chartName + "-namespace",
+	}
+
+	v, err := chartutil.ToRenderValues(c, valueMap, options, nil)
 	if err != nil {
-		logger.Warn("error encountered in CoalesceValues")
+		logger.Error("value rendering failed", zap.Any("values", v), zap.Error(err))
 		return iacDocuments, err
 	}
 
 	// render all files within the chart
 	var renderData map[string]string
-	renderData, err = engine.Render(c, v)
+	var e engine.Engine
+
+	// lint mode tells the rendering engine to be less strict when it comes to required variables
+	e.LintMode = true
+	renderData, err = e.Render(c, v)
 	if err != nil {
-		logger.Warn("error encountered while rendering chart", zap.String("template dir", templateDir))
+		logger.Warn("error encountered while rendering chart", zap.String("template dir", templateDir), zap.Error(err))
```

Comment:
I believe, this can be an `Error` level log?

Reply:
done